test/acceptance/test/smoke_tests.go

```diff
@@ -104,5 +90,43 @@ var _ = Describe("WEGO Acceptance Tests ", func() {
 			Eventually(string(session.Wait().Out.Contents())).Should(MatchRegexp(`-h, --help[\s]+help for wego`))
 			Eventually(string(session.Wait().Out.Contents())).Should(MatchRegexp(`-v, --verbose[\s]+Enable verbose output`))
 		})
+
+	}
+
+	It("Verify that wego help flag prints the help text", func() {
+
+		By("When I run the command 'wego --help' ", func() {
+			command := exec.Command(WEGO_BIN_PATH, "--help")
+			session, err = gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+
+		VerifyUsageText()
+
+	})
+
+	It("Verify that wego command prints the help text", func() {
+
+		By("When I run the command 'wego'", func() {
+			command := exec.Command(WEGO_BIN_PATH)
+			session, err = gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+
+		VerifyUsageText()
+
+	})
+
+	It("Verify that wego flux can print out version information", func() {
+
+		By("When I run 'wego flux -- -version", func() {
```

Comment:
would be good if this is changed to `wego flux -- -v` as that's what the test case is testing.